plugins/cilium-docker/driver/driver.go

```diff
@@ -410,6 +424,15 @@ func (driver *driver) joinEndpoint(w http.ResponseWriter, r *http.Request) {
 		DisableGatewayService: true,
 	}
 
+	// FIXME? Having the following code results on a runtime error: docker: Error
+	// response from daemon: oci runtime error: process_linux.go:334: running prestart
+	// hook 0 caused "exit status 1: time=\"2016-10-26T06:33:17-07:00\" level=fatal
+	// msg=\"failed to set gateway while updating gateway: file exists\" \n"
+	//
+	// If empty, it works as expected without docker runtime errors
+	// if rep.IPAMConfig.IP4 != nil {
+	//	res.Gateway = rep.IPAMConfig.IP4.Gateway.String()
+	// }
```

Comment:
Be aware that is the IPv4 Gateway. I was trying to add it on that field so it would appear filled in docker inspect

```
"NetworkID": "3c3c89a7ac132a1e33a45df53f85cde563976ed040ff232e1031f46c1951b1e8",
"EndpointID": "3ee074974e7dc2e58d066feb2acfc350398cb45eb8ea3c9284b20db561cc5ebf",
"Gateway": "",
"IPAddress": "10.1.28.238",
"IPPrefixLen": 32,
"IPv6Gateway": "f00d::c0a8:210b:0:0",
"GlobalIPv6Address": "f00d::c0a8:210b:0:8ad6",
"GlobalIPv6PrefixLen": 128,
"MacAddress":
```